app/helpers/application_helper.rb

```diff
@@ -7,15 +7,6 @@ def twitter_login_path
     "/auth/twitter"
   end
 
-  def current_user
-    return @current_user unless @current_user.nil?
-
-    return nil unless session.key?(:user_id)
-
-    @current_user = User.find_by_id(session[:user_id])
-    return @current_user
-  end
-
   def logged_in?
     return !current_user.nil?
   end
```

Comment:
My preferred style would be to delete this method, and when you need to find out, test either `if current_user` for presence, or `current_user.blank?` for absence. This saves the extra code, it's pretty readable inline